d-rita/fast-food-fast-react
# chore(redux): Set up redux

**What does this PR do?**
This PR sets up redux in the project.

**Description of tasks to be completed**
- Install redux, react-redux, redux-thunk
- Set up store
- Set up redux-devtools-extension

**How to manually test this**
- Fetch and check out this branch using `git fetch origin ch-set-up-redux-164353006 && git checkout ch-set-up-redux-164353006`
- To install dependencies, run yarn install
- In the terminal, use yarn start to run the application on localhost.
- Go to the browser and right-click, select` Inspect` and go to `Redux` to see the store.

**Related Pivotal Tracker Story**
[#164353006](https://www.pivotaltracker.com/story/show/164353006)
